### PR TITLE
Feat/lesson 06

### PR DESCRIPTION
**Pull request by sinkevichdenis**

### CODE REVIEW

File: app/middlewars/authMiddleware.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import config from '../config';
+import jwt from 'jsonwebtoken';
+
+export const authMiddleware = (req, res, next) => {
+    if (req.path === '/login') return next();
```
Comment (Ivanko5417) [inline app/middlewars/authMiddleware.js:5]: it's better to use this middleware after public routes middlewares

File: app/routes/auth.routes.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { login } from '../services/utils.service';
+
+// console.log('TOKEN LOGIN', await login('DenisUpdated', '66699'))
```
Comment (Ivanko5417) [inline app/routes/auth.routes.js:3]: let's remove unnecessary comments :)

**Review by Ivanko5417 — APPROVED** [review]

File: app/middlewars/authMiddleware.js (hunk 1 above)
Comment (sinkevichdenis) [reply app/middlewars/authMiddleware.js:5]: done

File: app/routes/auth.routes.js (hunk 2 above)
Comment (sinkevichdenis) [reply app/routes/auth.routes.js]: done

File: app/app.js [hunk 3]
```diff
@@ -13,8 +13,8 @@ process.on('uncaughtException', logErrorHandler);
 
 app.use(cors());
 app.use(express.json({ extended: true }));
-app.use(authMiddleware);
 app.use('/', router);
+app.use(authMiddleware);
```
Comment (Ivanko5417) [inline app/app.js:17]: you have public route: '/login'
all other routes are private and should be validated by authMiddleware
you can do like this
```javascript
  app.use(publicRoutes);
  app.use(authMiddleware);
  app.use(priveteRoutes);
```
publicRoutes has login routes
all other routes are included into privateRoutes

**Review by Ivanko5417 — CHANGES_REQUESTED** [review]

File: app/app.js (hunk 3 above)
Comment (sinkevichdenis) [reply app/app.js]: done

**Review by Ivanko5417 — APPROVED** [review]

**Pull request merged**